client/src/components/Member/Member.tsx

```diff
@@ -22,8 +22,8 @@ const StyledMember = styled.section`
 `;
 
 interface MemberProps {
-  user: IMember;
-  onActionClick?: (useId?: IMember) => void;
+  user: RoomMemberFragment;
```

Comment:
Since i used fragments that's why the typescript typing were not matching, now fixed!